### PR TITLE
Medical guidebook entries

### PR DESCRIPTION
**Pull request by MagnusCrowe**
## About the PR

This PR adds several guidebook entries to help players understand the mechanics behind the medical profession, with helpful coloring and even pictures! Issue https://github.com/space-wizards/space-station-14/issues/13506

Includes cryogenic information that might interest issue https://github.com/space-wizards/space-station-14/issues/13512.

I was going to add a chemistry entry here too but I could not come up with a good way to have the entry automatically update chemical formulas.  If we put any recipes in the guide, and the recipe or values change it could be a nightmare of conflicting information and tedious vigilant updates to the guidebook.  If someone know how to do this and wants to tell me, I'll add a chemistry page too.

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: MagnusCrowe
- add: Added guidebook entries for Medical

### CODE REVIEW

**Comment by moonheart08** [comment]
@MagnusCrowe The hard part of the chemistry guidebook is already done in one of my PRs, #14060. What needs done still is wiring up the backend to the frontend ui of the guidebook.

**Comment by moonheart08** [comment]
Also the usual headsup: If you take my work, please use git merge to incorporate it, not copy paste.

**Comment by MagnusCrowe** [comment]
> Also the usual headsup: If you take my work, please use git merge to incorporate it, not copy paste.

It won't be part of this pull request but if I am the one to write the Chem guide, I'll do that.

**Comment by MagnusCrowe** [comment]
Any chance of a review?

File: Resources/Prototypes/Guidebook/medical.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  
+- type: guideEntry
+  id: CryoMedicine
+  name: guide-entry-cryo-medicine
+  text: "/Server Info/Guidebook/Medical/DiagnosisAndTreatment/CryoMedicine.xml"
+
+- type: guideEntry
+  id: CloningAndMorgue
+  name: guide-entry-cloning-and-morgue
+  text: "/Server Info/Guidebook/Medical/CloningAndMorgue.xml"
```
Comment (metalgearsloth) [inline Resources/Prototypes/Guidebook/medical.yml:73]: Please alphabetise these, at the very least with the child entries.

File: Resources/Prototypes/Guidebook/shiftandcrew.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@
   name: guide-entry-jobs
   text: "/Server Info/Guidebook/Jobs.xml"
   children:
+  - Medical
   - Botany
   - Engineering
   - Science
```
Comment (metalgearsloth) [inline Resources/Prototypes/Guidebook/shiftandcrew.yml:9]: Same here please keep it alphabetical and not just inserting it on top.

File: Resources/Server Info/Guidebook/Medical/CloningAndMorgue.xml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+It is enevitable that crew and passengers will die onboard the station.  Through the power of cloning, they can be reborn!
+
+## Diagnosing death
+It is obvious when a patient is dead. What is less obvious is whether or not they can be cloned.
+
+If the corpse has an examination message like the one below, they are NOT clonable. This is sometimes refered to as being a husk or being soulless.
+- [color=Yellow]Their soul has departed and moved on. Any recovery is unlikely.[/color]
+
+If the corpse has an examination message like the one below, they ARE still clonable.
+- [color=Red]Their soul has departed.[/color]
```
Comment (metalgearsloth) [inline Resources/Server Info/Guidebook/Medical/CloningAndMorgue.xml:12]:
```suggestion
If the corpse has an examination message like the one below, they ARE still cloneable, but their "soul" (read: player) is not present.
- [color=Red]Their soul has departed.[/color]
```

File: Resources/Server Info/Guidebook/Medical/CloningAndMorgue.xml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+</Box>
+
+The basic steps of cloning are as follows:
+- Place the clonable corpse into the Medical Scanner
+- Interact with the Cloning Console, click the clone button
+- Wait for the patient to accept the cloning prompt and exit the Cloning Pod
+
+This can only be accomplished if there is enough biomass inside the Cloning Pod.  If the Cloning Console gives you an error, requesting more biomass, you will need to use the Biomass Reclaimer to convert soulless corpses into biomass.  After the reclaimer is done grinding up the body, you can pick up the biomass and put it in the Cloning Pod by clicking on it with the biomass.
+
+This biomass reclamation process usually occures in advance.  You will want to reclaim biomass from any corpse that is unclonable.  The reclaimer is reletively slow and often times you will find your clone room looking more like a corpse pile.  Reclaim early and often.
```
Comment (metalgearsloth) [inline Resources/Server Info/Guidebook/Medical/CloningAndMorgue.xml:37]:
```suggestion
This biomass reclamation process usually occurs in advance.  You will want to reclaim biomass from any corpse that is uncloneable.  The reclaimer is relatively slow and often times you will find your clone room looking more like a corpse pile.  Reclaim early and often.
```

File: Resources/Server Info/Guidebook/Medical/DiagnosisAndTreatment/CryoMedicine.xml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+As long as the pod has power, it will continuously inject the patient with the [color=#0091ff]Cryoxadone[/color]. If healing stops, you may have run out of medicine. If they are not healing even with [color=#0091ff]Cryoxadone[/color] in the pod, you may need to strip them of any clothing that is keeping them warm in the pod or lower the temperature on your freezer. 
+
+You can monitor their status by clicking on the pod to show a real-time health analyzer.  Alt-clicking the pod will eject the beaker. 
+
+Once your patient is healed, you may eject them from the pod and begin the process of warming them up.  While in the pod, their body temperature will get very low, but the pod protects them.  Once they leave the pod, they will still be cold but no longer be protected by the pod.  To warm them up quickly the application of a low dose of [color=#ff7db5]Leporazine[/color] is reccommended.  If no [color=#ff7db5]Leporazine[/color] is on hand, you may need to rely on a winter jacket and scarf.
```
Comment (metalgearsloth) [inline Resources/Server Info/Guidebook/Medical/DiagnosisAndTreatment/CryoMedicine.xml:24]:
```suggestion
Once your patient is healed, you may eject them from the pod and begin the process of warming them up.  While in the pod, their body temperature will get very low, but the pod protects them.  Once they leave the pod, they will still be cold but no longer be protected by the pod.  To warm them up quickly the application of a low dose of [color=#ff7db5]Leporazine[/color] is recommended.  If no [color=#ff7db5]Leporazine[/color] is on hand, you may need to rely on a winter jacket and scarf.
```

File: Resources/Server Info/Guidebook/Medical/DiagnosisAndTreatment/Damage/Toxin.xml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+# Toxin
+<Box>
+[color=Gold]Toxin[/color]:
+- [color=DeepSkyBlue]Poison[/color]
+- [color=DeepSkyBlue]Radiation[/color]
+</Box>
+Toxin damage is the group of damage your patients take from injesting poisons, overdosing, and being exposed to radioactive materials.
+
+## Treatment
+All of the cures to toxin damage target the [color=DeepSkyBlue]damage types[/color], not the entire [color=Gold]damage group[/color] .  This means that radiation and poison will have completely different treatments.
```
Comment (metalgearsloth) [inline Resources/Server Info/Guidebook/Medical/DiagnosisAndTreatment/Damage/Toxin.xml:11]:
```suggestion
All of the cures to toxin damage target the [color=DeepSkyBlue]damage types[/color], not the entire [color=Gold]damage group[/color].  This means that radiation and poison will have completely different treatments.
```

File: Resources/Server Info/Guidebook/Medical/DiagnosisAndTreatment/Diagnosis.xml [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+<Box>[color=bisque]They look pale.[/color]</Box>
+
+Details on these messages and how to decode them can be found in the Damage Types section.
+
+## Medical HUD
+<GuideEntityEmbed Entity="ClothingEyesHudMedical" Caption="Medical HUD"/>
+The medical HUD is [color=red]not[/color] yet fully implemented, but when it is, it should provide at-a-glance diagnostic information about the patient's overall health.  
+
+Right now it is just a cosmetic item.
+</Document>
```
Comment (metalgearsloth) [inline Resources/Server Info/Guidebook/Medical/DiagnosisAndTreatment/Diagnosis.xml:34]: Remove

File: Resources/Server Info/Guidebook/Medical/DiagnosisAndTreatment/Viruses.xml [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+<GuideEntityEmbed Entity="Vaccine" Caption="Vaccine"/>
+</Box>
+
+## Curing the disease
+As mentioned in "Identify the virus", you can produce a diagnosis report that will tell you the cure.  Usually a short period of sleep is enough to cure most diseases but more exotic diseases may require special methods that may reach even beyond your capability as a doctor to cure.  For example, the AMIV virus can be easily cured by a banana infused cocktail from the bar called Banana Honk.
+
+Alternatively, you can treat diseases with [color=#9942f5]Spaceacillin[/color].  Be careful however, [color=#9942f5]Spaceacillin[/color] causes cellular damage also known as [color=red]space cancer[/color].  See the genetic damage guide to learn more about cellular damage.  If you do choose to administer [color=#9942f5]Spaceacillin[/color], low doses can be given in succession until a disease is cured to minimize cellular damage (sometimes 5 units is all you need).
+
+Once the disease is cured, the patient is immune to catching that disease again.  They can still catch other diseases, but the one you cure is no longer a worry.
+</Document>
```
Comment (metalgearsloth) [inline Resources/Server Info/Guidebook/Medical/DiagnosisAndTreatment/Viruses.xml:50]: Diseases are gonna get removed until they get refactored so remove.

File: Resources/Server Info/Guidebook/Medical/Medical.xml [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+</Box>
+<Box>
+<GuideEntityEmbed Entity="DiseaseSwab" Caption="Sterile Swab"/>
+<GuideEntityEmbed Entity="DiagnosisReportPaper" Caption="Diagnosis Report"/>
+<GuideEntityEmbed Entity="Vaccine" Caption="Vaccine"/>
+</Box>
+<Box>
+<GuideEntityEmbed Entity="ClothingHeadHatHoodBioVirology" Caption="Bio Hood"/>
+<GuideEntityEmbed Entity="ClothingOuterBioVirology" Caption="Bio Suit"/>
+</Box>
```
Comment (metalgearsloth) [inline Resources/Server Info/Guidebook/Medical/Medical.xml:88]: As above.

Comment (metalgearsloth) [inline Resources/Server Info/Guidebook/Medical/Medical.xml:88]: I think generally listing these is a bad idea as they're liable to change a lot.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Cloneable not clonable IMO. Docs also need spellcheck run over them.

**Comment by MagnusCrowe** [comment]
Thank you for all the feedback.  I will work on it.

**Comment by AJCM-git** [comment]
re-open once you come back

**Pull request closed without merging**